src/Config/Converters/DataSourceConverterFactory.cs

```diff
@@ -47,7 +47,7 @@ public DataSourceConverter(bool replaceEnvVar)
 
         public override DataSource? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-            DataSource dataSource = new(DatabaseType.MSSQL, string.Empty, null);
+            DataSource dataSource = new(DatabaseType.MSSQL, string.Empty, null, null);
```

Comment:
nit: name the parameters that are null for easier readability as to what you are passing along.